platform-sdk/docs/proposals/001-merkle-state-api/001-merkle-state-api.md

```diff
@@ -0,0 +1,255 @@
+# Merkle State API
+
+## Summary
```

Comment:
Your summary sounds more like a todo list to get the proposal implemented.

Reply:
How you you propose to reword it?

Reply:
Just an idea:

This proposal aims to define a public api for the ledger state. The api should be used to interact with the Merkle tree that will be an implementation detail in future. Since we already have a state api in the services layer a goal of the proposal is to move. that api to the platform layer and use it as central api to access the ledger state in all our code & products.

Reply:
I will modify the summary